internals/src/Tizen.Peripheral/Tizen.Peripheral/Adc.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeAdc = Interop.Peripheral.Adc;
+
+namespace Tizen.Peripheral
```

Comment:
CONSIDER using plural namespace names where appropriate.
```suggestion
namespace Tizen.Peripherals
```
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces

Reply:
I am aware that docs recommends to use:
```✔️ CONSIDER using plural namespace names where appropriate.```
but I want to follow naming convention in C language API and Tizen privilege name.